### ISSUE TITLE
Case-insensitive sorting

### ISSUE THREAD

**Issue by ArVan** [balderdashy/sails-mongo#155]
Hello everyone.
I know that mongodb performs sorting on string attributes in a **case-sensative** way. So when I sort some query results, I get like:
````javascript
{
  name: "A"
},
{
  name: "B"
},
{
  name: "a"
}
````
Some say **case-insensitive** sorting can be achieved by performing aggregation (http://stackoverflow.com/a/22931481/1065687).

Does sails adapter provide any solution on this issue?

**Comment by ArVan** [comment]
Hi everyone. Is there any news on case-insensitive sorting?

**Comment by sailsbot** [comment]
Thanks for posting, @ArVan.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
